core/interoperation/Cargo.toml

```diff
@@ -16,3 +16,13 @@ ckb-vm = { version = "0.21", features = ["aot"] }
 
 [target.'cfg(target_arch = "aarch64")'.dependencies]
 ckb-vm = { version = "0.21", features = ["asm"] }
+
+[dev-dependencies]
+hex = "0.4"
```

Comment:
The hex codec function is defined in https://github.com/nervosnetwork/axon/blob/main/protocol/src/codec/mod.rs#L53:L68